.github/workflows/release-snapshot.yml

```diff
@@ -31,6 +31,10 @@ jobs:
           version: 9.4.0
           run_install: false
 
+      - name: Add npm auth token to pnpm
+        run: pnpm config set '//registry.npmjs.org/:_authToken' "${NPM_TOKEN_ELEVATED}"
+        env:
+          NPM_TOKEN_ELEVATED: ${{secrets.NPM_TOKEN_ELEVATED}}
```

Comment:
Whats this for?

Reply:
this was to publish the package in canary to test for @dvoytenko 

not relevant to the PR